src/Nancy/Json/Simple/SimpleJson.cs

```diff
@@ -1237,6 +1238,25 @@ interface IJsonSerializerStrategy
         string MapDictionaryKeyToFieldName(string stringKey);
     }
 
+#if SIMPLE_JSON_INTERNAL

```

Comment:
I'm not sure what it's used for, I just noticed it wasn't included like on the existing classes so I pointed it out incase it was needed for something.